specification/visualstudio/resource-manager/Microsoft.VisualStudio/preview/2018-08-01-preview/Projects.json

```diff
@@ -0,0 +1,717 @@
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "repository": {
+          "description": "Code repository where the application source resides.",
+          "$ref": "#/definitions/CodeRepository"
+        }
+      },
+      "required": [
+        "name",
+        "templateId"
+      ]
+    },
+    "Resource": {
+      "description": "A generic Azure Resource Manager resource.",
+      "x-ms-azure-resource": true,
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Unique identifier of the resource.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "location": {
+          "description": "Resource location.",
+          "type": "string"
+        },
+        "name": {
+          "description": "Resource name.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "tags": {
+          "description": "Resource tags.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "type": {
+          "description": "Resource type.",
+          "readOnly": true,
+          "type": "string"
+        }
+      }
+    },
+    "ProjectResource": {
+      "description": "A Visual Studio Team Services project resource.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "kind": {
+          "description": "Kind of project resource",
+          "type": "string",
+          "enum": [
+            "project",
+            "bootstrappedProject"
+          ],
+          "x-ms-enum": {
+            "name": "ProjectResourceKind",
+            "modelAsString": true
+          }
+        },
+        "properties": {
+          "description": "Custom properties of project resource.",
+          "$ref": "#/definitions/ProjectResourceProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "ProjectResourceProperties": {
+      "description": "Defines the custom properties of project resource.",
+      "type": "object",
+      "properties": {
+        "processTemplateId": {
+          "description": "Process template to use in the project.",
+          "type": "string",
+          "enum": [
+            "Scrum",
+            "Agile",
+            "Cmmi"
+          ],
+          "x-ms-enum": {
+            "name": "ProcessTemplate",
+            "modelAsString": true
+          }
+        },
+        "versionControlOption": {
+          "description": "Version control to use for the default repo created in the project.",
+          "type": "string",
+          "enum": [
+            "Git",
+            "Tfvc"
+          ],
+          "x-ms-enum": {
+            "name": "VersionControlOption",
+            "modelAsString": true
+          }
+        },
+        "ownerUpn": {
```

Comment:
[nit] I believe the linter should let you name this `ownerUPN`, but I could be wrong.

Reply:
Server side we already have ownerUpn, so this is fine.